src/screens/CollectionEditScreen.tsx

```diff
@@ -114,11 +114,24 @@ export default function CollectionEditScreen(props: PropsType) {
       }
 
       await dispatch(collectionUpload(colMgr, collection));
-      dispatch(pushMessage({ message: "Collection saved", severity: "success" }));
-      navigation.goBack();
-      // FIXME having the sync manager here is ugly. We should just deal with these changes centrally.
-      const syncManager = SyncManager.getManager(etebase);
-      dispatch(performSync(syncManager.sync())); // not awaiting on puprose
+      if (colUid.length > 0) {
+        dispatch(pushMessage({ message: "Notebook saved", severity: "success" }));
+        navigation.goBack();
+      } else {
+        dispatch(pushMessage({ message: "Notebook created", severity: "success" }));
+        const previousScreen = navigationState.name;
```

Comment:
I added a button in the NotePropertiesScreen to be able to create a new notebook from a new note. 
In that case, when the collection has been created, we need to go back to the NotePropertiesScreen with the new colUid so the Select is properly set. I tried and you can't pop then replace (because the screen is unmounted at pop), so you need to replace first and then pop. I also tried navigate that's supposed to go back to the named screen in the stack but somehow it triggers a new stack so we end up at the Home screen…

Reply:
Ah, that confused me. Please try to avoid having unrelated changes in the same commit as it makes reviewing significantly harder. This review for example has been much harder than all of the rest of the PRs combined.

Reply:
I thought it was related because that's a feature of NotePropertiesScreen

Reply:
OK, if it's required for the change, no need to separate. It's just that things that are fixed along the way that are not a good addition. This felt to me like an unrelated improvement.

Reply:
There is one unrelated improvement though with removing performSync and changing the reducers. I'll separate it.

Reply:
Done?